Jantimizer/QueryTestSuite/Connectors/MySql.cs

```diff
@@ -0,0 +1,135 @@
+﻿using MySqlConnector;
```

Comment:
Could it make sense for this to called 'MySqlConnector' rather than just 'MySql'?